saml2idp/views.py

```diff
@@ -18,6 +19,13 @@
 from . import registry
 from . import xml_signing
 
+# The 'schemes' argument for the URLValidator was introduced in Django 1.6. This
+# ensure that URL validation works in 1.5 as well.
+try:
+    URL_VALIDATOR = URLValidator(schemes=('http', 'https'))
```

Comment:
Interesting suggestion, I haven't really thought about custom schemes. Unfortunately, the Django URL validator only supports whitelisted schemes, so custom ones would have to be explicitly defined ([`django.core.validators:107`](https://github.com/django/django/blob/master/django/core/validators.py#L107). For now, I wouldn't want to support it until we get to a point where this is a requirement for SLO redirects.